app/Domain/Timesheets/Controllers/ShowMyList.php

```diff
@@ -43,18 +45,20 @@ public function run(): Response
             $kind = ($_POST['kind']);
         }
 
-        $dateFrom = Carbon::now($_SESSION['usersettings.timezone'])->setTimezone('UTC')->startOfMonth();
+        // Use UTC here as all data stored in the database should be UTC (start in user's timezone and convert to UTC).
+        // The front end javascript is hardcode to start the week on mondays, so we use that here too.
+
+        //Get start of the week in current users timezone and then switch to UTC
+        $dateTimeHelper = new DateTimeHelper();
+        $dateFrom = $dateTimeHelper->userNow()->startOfMonth();
+        $dateTo = $dateTimeHelper->userNow()->endOfMonth();
```

Comment:
Should these be converted to UTC, so only the template layer uses users timezone?

Reply:
You are correct. Thanks for catching that, those should have been UTC